### PR TITLE
add file index to pass

### PR DESCRIPTION
**Pull request by nozomi-link**
add file index to pass when tap the image

### CODE REVIEW

**Review by dondoko-susumu — COMMENTED** [review]
It necessary type of argument in `onPress`.

https://github.com/dondoko-susumu/react-native-card-media/pull/8/files#diff-e9f4bf08e954f04bdccce814975b3abcc0d021a450e8bc87aeff70cf5b3f7b53L22

File: components/CardMedia.js [hunk 1]
```diff
@@ -67,122 +67,148 @@ export default class CardMedia extends Component<Props> {
...
-              <Text
-                style={titleStyle}
-                numberOfLines={1}
-                ellipsizeMode={'tail'}
-              >
-                {title}
-              </Text>
-            </View>
-          </TouchableWithoutFeedback>
+          <View style={styles.overlay} pointerEvents='none'>
```
Comment (dondoko-susumu) [inline components/CardMedia.js:168]: Why remove `TouchableWithoutFeedback`?

**Review by dondoko-susumu — COMMENTED** [review]
Hi!, @nozomi-link.
Thanks for the great PR.
I have reviewed your PR.
I commented on a few points.
Please confirm it.

And by the way, I hope you'll update the README.

File: components/CardMedia.js (hunk 1 above)
Comment (nozomi-link) [reply components/CardMedia.js:168]: 透過させているタイトル背面の画像をタップできるようにした為、不要になったからです。

Comment (dondoko-susumu) [reply components/CardMedia.js:168]: You are Japanese! OK.
I like Japan too.
But, English, please, in here.

Comment (nozomi-link) [reply components/CardMedia.js:168]: it is no longer need because you can now tap images behind title.

Comment (dondoko-susumu) [reply components/CardMedia.js:168]: I see.
Then, `titleTouchable` option is no longer necessary.
And, remove it too.

**Review by dondoko-susumu — COMMENTED** [review]
`Pending` ?
I do not know how to use GitHub very well.

**Comment by nozomi-link** [comment]
i removed titleTouchable option and updated README.
Please confirm it.

File: components/CardMedia.js (hunk 1 above)
Comment (nozomi-link) [reply components/CardMedia.js:168]: i removed titleTouchable option and updated README.
Please confirm it.

Comment (dondoko-susumu) [reply components/CardMedia.js:168]: thanks.
Add description about arguments to onPress in README.

Comment (nozomi-link) [reply components/CardMedia.js:168]: i finished editing it.

Comment (dondoko-susumu) [reply components/CardMedia.js:168]: Thanks.
It necessary type of argument in onPress.

👇
https://github.com/dondoko-susumu/react-native-card-media/blob/56a47f64885a0060b3f461b88115bd5aceff4993/components/CardMedia.js#L22

Comment (nozomi-link) [reply components/CardMedia.js:168]: i added necessary type of argumente in onPress.
pls confirm it.

thank you

File: components/CardMedia.js [hunk 2]
```diff
@@ -19,10 +19,9 @@ type Props = {
   title: string,
   showTitle:boolean,
   titleStyle?: StyleObj,
-  onPress?: () => mixed,
+  onPress?: (fileIndex:Number) => mixed,
```
Comment (dondoko-susumu) [inline components/CardMedia.js]: Number -> number

**Pull request merged**